docs/feature_pointing_device.md

```diff
@@ -731,3 +732,598 @@ In general the following two functions must be implemented in appropriate locati
+
+```c
+// in config.h:
+#define POINTING_DEVICE_MODES_ENABLED
+```
+
+###  Activating Pointing Device Modes
+The first 15 pointing device modes can easily be activated by keypress by adding the following macros to a keymap by using the following keycode macros:  
+
+#### Keycode Macros (_for _`PM_NONE`_ and the first 15 modes only_)
+| Keycode Macro  | Description                                                                                                           |
+| -------------- | --------------------------------------------------------------------------------------------------------------------- |
+| `PM_MO(<pm>)`  | Momentary key for pointing mode `<pm>` (i.e active while key pressed deactivate on release)                           |
+| `PM_TG(<pm>)`  | Toggle key for pointing mode `<pm>` (toggle on release, remain until pressed and released again)                      |
+ 
+***Note: For pointing device modes above mode id 15 a custom keycode would need to be added unless the mode is being activated through some other means (such as on specific layers see advanced use below)***
+
+#### Toggled Pointing Device Modes vs Momentary Pointing Modes
+Pointing device modes activated by toggle type functions/macros have their mode id saved until toggled off or a different mode is activated by toggle overwriting the last toggle mode.  When a Momentary type function or key is used while another mode is toggled the toggled mode will be reactivated once the momentary mode is released. Toggling a mode on will overwrite both the saved toggled mode id (_if different_) as well as the current mode id while using a momentary type key will only overwrite the current mode.
+
+#### Built-in Pointing Device Modes
+| Pointing Device Mode  | Alias     | Mode Id | Description                                                                                                                                                 |
+| :-------------------- | --------- | :-----: | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `PM_NONE`             |  _None_   |     0   | Null pointing mode that will will pass through normal x and y output of pointing device (Cannot be overwritten)                                             |
+| `PM_DRAG`             | `PM_DRG`  |     1   | Change x and y movement of pointing device into h and v axis values  `x->h`  `y->v`                                                                         |
+| `PM_PRECISION`        | `PM_PRE`  |     2   | Reduce x and y movement output of pointing device by the divisor which can affect other modes when toggled (_see notes_)                                    |
```

Comment:
Is the mode_id numbering still accurate? In the most recent commit I think PM_PRECISION and PM_DRAG switched cases

Reply:
Argh, yeah you are correct, Thank you!
Fixed in latest commit this is the last time I reorder them I just decided that because of Precision modes special handling I should discourage overwriting it.